test/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -44,39 +51,91 @@ class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec {
 
       val request =
         MovementRequest("eori", "provider-id", MovementType.Arrival, ConsignmentReferences("ref", "value"), MovementDetailsRequest("datetime"))
-      await(connector.submit(request))
+      connector.submit(request).futureValue
 
       verify(
-        postRequestedFor(urlEqualTo("/movements"))
+        postRequestedFor(urlEqualTo(MovementsSubmissionEndpoint))
           .withRequestBody(
             equalTo(
-              "{\"eori\":\"eori\",\"providerId\":\"provider-id\",\"choice\":\"EAL\",\"consignmentReference\":{\"reference\":\"ref\",\"referenceValue\":\"value\"},\"movementDetails\":{\"dateTime\":\"datetime\"}}"
+              """{"eori":"eori","providerId":"provider-id","choice":"EAL","consignmentReference":{"reference":"ref","referenceValue":"value"},"movementDetails":{"dateTime":"datetime"}}"""
             )
           )
       )
     }
   }
 
   "Submit Consolidation" should {
-    given(config.customsDeclareExportsMovements).willReturn(downstreamURL)
 
     "POST to the Back End" in {
       stubFor(
-        post("/consolidation")
+        post(ConsolidationsSubmissionEndpoint)
           .willReturn(
             aResponse()
               .withStatus(Status.ACCEPTED)
           )
       )
 
       val request = DisassociateDUCRRequest("pid", "eori", "ucr")
-      await(connector.submit(request))
+      connector.submit(request).futureValue
 
       verify(
-        postRequestedFor(urlEqualTo("/consolidation"))
-          .withRequestBody(equalTo("{\"providerId\":\"pid\",\"eori\":\"eori\",\"ucr\":\"ucr\",\"consolidationType\":\"DISASSOCIATE_DUCR\"}"))
+        postRequestedFor(urlEqualTo(ConsolidationsSubmissionEndpoint))
+          .withRequestBody(equalTo("""{"providerId":"pid","eori":"eori","ucr":"ucr","consolidationType":"DISASSOCIATE_DUCR"}"""))
+      )
+    }
+  }
+
+  "fetch all Submissions" should {
+
+    "send GET request to the backend" in {
+
+      val submissionJson = Json.toJson(Seq(exampleSubmissionFrontendModel()))
```

Comment:
Similar to above, these tests should act as contract tests. So if we are generating the JSON using our own models, the test doesn't really provide much value. It's the reason the above have some JSON strings in the assertions